cid/common.py

```diff
@@ -361,16 +390,32 @@ def _deploy(self, dashboard_id: str=None, recursive=True, update=False, **kwargs
         # TODO: check if datasets returns explicit permission denied and only then discover dashboards as a workaround
         self.qs.discover_dashboards()
 
+        dashboard_id = dashboard_id or get_parameters().get('dashboard-id')
+        if not dashboard_id:
+            standard_categories = ['Foundational', 'Advanced', 'Additional']
+            all_categories = set([f"{dashboard.get('category', 'Custom')}" for dashboard in self.resources.get('dashboards').values()])
+            non_standard_categories = [cat for cat in all_categories if cat not in standard_categories]
+            categories =  standard_categories + sorted(non_standard_categories)
+            dashboard_options = {}
+            for category in categories:
+                dashboard_options[f'{category.upper()}'] = '[category]'
+                for dashboard in self.resources.get('dashboards').values():
+                    if dashboard.get('category', 'Custom') == category:
```

Comment:
Let's change to Other or Uncategorized. Custom has additional context which might confuse customers